src/applications/personalization/dashboard/components/debts-v2/DebtsCardV2.jsx

```diff
@@ -48,19 +46,18 @@ export const DebtsV2 = ({ debts }) => {
         data-testid="debt-card-v2"
       >
         <h3 className="vads-u-margin-top--0" data-testid="debt-total-header-v2">
-          ({currency(totalDebt)})
-        </h3>
-        <h4 className="vads-u-margin-top--0">
           {debtsCount} overpayment debt
           {debtsCount > 1 ? 's' : ''}
-        </h4>
+        </h3>
         <p className="vads-u-margin-bottom--1 vads-u-margin-top--0">
-          Last updated {formattedLastUpdatedDate}
+          Updated on{' '}
+          {format(new Date(formattedLastUpdatedDate), 'MMMM dd, yyyy')}

```

Comment:
I might be missing something, but this looks like it is being formatted twice, but with the same format? Is that needed?

Reply:
yep. I'm dumb. lol Fixed. Thanks for catching that